Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -641,8 +641,8 @@ public class HTTPClient {
         public var redirectConfiguration: RedirectConfiguration
         /// Default client timeout, defaults to no timeouts.
         public var timeout: Timeout
-        /// Timeout of pooled connections
-        public var maximumAllowedIdleTimeInConnectionPool: Optional<TimeAmount>
+        /// Connection pool configuration.
+        public var poolConfiguration: PoolConfiguration
```

Comment:
We cannot remove fields in semver minors. The old field must continue to exist, though it's acceptable for it to be computed from the new field.

Reply:
That field was never in the released version, only in master

Reply:
Good catch, you're right.